cmd/habitat/procs/proc.go

```diff
@@ -20,7 +20,8 @@ type Proc struct {
 	cmd     *exec.Cmd
 	errChan chan ProcError
 
-	config *configuration.App
+	config  *configuration.App
+	stopped bool
```

Comment:
why did this get added? is it to be able to restart a process?

Reply:
Marking it as stopped lets us discard errors that are caught from the process exiting.